scripts/build_rln.sh

```diff
@@ -17,7 +17,7 @@ fi
 host_triplet=$(rustup show | grep "Default host: " | cut -d' ' -f3)
 
 # Download the prebuilt rln library if it is available
-if curl --silent --fail-with-body -L "https://github.com/vacp2p/zerokit/releases/download/v0.3.1/$host_triplet-rln.tar.gz" >> "$host_triplet-rln.tar.gz"
+if curl --silent --fail-with-body -L "https://github.com/vacp2p/zerokit/releases/download/v0.3.2/$host_triplet-rln.tar.gz" >> "$host_triplet-rln.tar.gz"
```

Comment:
Maybe an unnecessary overkill, but what about getting the version from the submodule?

Something like 

```
$ git submodule status vendor/zerokit | awk '{print $3}' | sed 's/[()]//g'
v0.3.1
``` 

So that there is only one source of truth for the version?

Reply:
will address in a follow up pr, thanks!